src/layouts/default/PlayerOSD/Player.vue

```diff
@@ -1,8 +1,5 @@
 <template>
-  <div
-    :class="`${useFloatingPlayer ? 'mediacontrols-bg-2' : 'mediacontrols-bg-1'}`"
-    :style="`background: ${backgroundColor};`"
-  ></div>
+  <div class="mediacontrols-bg" :data-floating="useFloatingPlayer"></div>
```

Comment:
data-floating is a bit of a weird property name

Reply:
I'm using the `data-*` to reflect the state in the HTML ([Cube CSS](https://cube.fyi/exception.html#why-data-attributes) does the same thing). We could use a class here and do something like `class="mediacontrols-bg mediacontrols-bg__floating"`, but I find the data attribute a little clearer.